lib/ansible/plugins/inventory/yaml.py

```diff
@@ -45,7 +45,7 @@
             children:
                 group_x:
                     hosts:
-                        test5
+                        test5:
```

Comment:
Could you please add an additional group with two hosts so that it will be more clear how to use two or more hosts with colon ? Rest LGTM

Reply:
I added a group and some comments, is that ok ?